### PR TITLE
Pussy time pussy clean pussy fresh

### PR DESCRIPTION
**Pull request by kevadesu**
first of all, you could've written Follow our Twitter, would've been simpler right? Then, a card has no face. sure it can be ugly or beautiful, but is it a living thing? 💀 I suggest you remove the word "face". better right? good. it was nice doing business with you.

### CODE REVIEW

**Comment by kevadesu** [comment]
Merge be when? 👀

**Comment by locness3** [comment]
Thank you for the pull request!

> you could've written Follow our Twitter

Better to say what to expect from the Twitter

> Then, a card has no face.

You might be right, maybe a more appropriate word would be "side". If you're going to change this, you could change the relevant markup and style as well. Btw this is unused code.

> but is it a living thing

Our projects are currently alive and that's a good thing \:)

> it was nice doing business with you.

Wouldn't say we're doing business, we're just having fun

**Comment by kevadesu** [comment]
> Thank you for the pull request!
> 
> > you could've written Follow our Twitter
> 
> Better to say what to expect from the Twitter
> 
> > Then, a card has no face.
> 
> You might be right, maybe a more appropriate word would be "side". If you're going to change this, you could change the relevant markup and style as well. Btw this is unused code.
> 
> > but is it a living thing
> 
> Our projects are currently alive and that's a good thing \:)
> 
> > it was nice doing business with you.
> 
> Wouldn't say we're doing business, we're just having fun

The "doing business" part was just a joke

**Comment by locness3** [comment]
If you'd like to merge this, please :

- Revert the Twitter change
- change parts of markup and stylesheets that uses the word "face" to use "side"

**Comment by kevadesu** [comment]
> If you'd like to merge this, please :
> 
> - Revert the Twitter change
> - change parts of markup and stylesheets that uses the word "face" to use "side"

Very well. One second

**Comment by kevadesu** [comment]
done

**Comment by locness3** [comment]
I'm still seeing `face` classes in the HTML and in the relevant SCSS file. \:)

**Comment by kevadesu** [comment]
> I'm still seeing `face` classes in the HTML and in the relevant SCSS file. :)

oops lemme fix

**Comment by kevadesu** [comment]
done

File: assets/css/style.css [hunk 1]
```diff
@@ -1 +1 @@
...
...k:focus,.block-link:active{transform:scale(1.1)}.block-link::after{content:"→";display:inline-block;margin-left:.5em;transition:.2s}.block-link:hover::after,.block-link:focus::after,.block-link:active::after{transform:translateX(10px)}.block-link svg{display:inline-block;vertical-align:middle;margin-right:.2em}.block-links{list-style-type:none;margin:0;padding:0}.block-links li{display:inline-block;margin:1em 15px}.projects{text-align:center}.projects *{text-align:left}.project,.project-twosided{display:inline-block;position:relative;min-width:200px;max-width:335px;margin:15px;vertical-align:top;perspective:800px}.project,.project-twosided .side{padding:15px;color:#fff;border:1px solid #fff;background-color:#000;border-radius:5px}.project.block-link::after{display:block;text-align:right}.project .project-logo{margin-bottom:.3em;transition:.2s}.project.block-link:hover .project-logo,.project.block-link:focus .project-logo,.project.block-link:active .project-logo{transform:scale(0.9)}.project.uverse{background-color:#008663;border-color:#008663}.project-status{text-transform:uppercase;font-size:.9em;opacity:.9}.project-status em{font-style:normal}/*# sourceMappingURL=style.css.map */
```
Comment (locness3) [inline assets/css/style.css:1]: @kevadesu please read assets/css/README.txt

**Review by locness3 — CHANGES_REQUESTED** [review]

**Comment by locness3** [comment]
Please consider closing this pull request or making the requested changes

**Comment by kevadesu** [comment]
yeah ok sure I'll close it

**Pull request closed without merging**